gradle.properties

```diff
@@ -15,18 +15,15 @@ org.gradle.parallel.threads=8
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 org.gradle.parallel=true
-org.gradle.java.home=/Applications/Android Studio.app/Contents/jre/jdk/Contents/Home
 # ---------------------------------- MiniApp SDK flags ------------------------------------
 # Artifactory information needed.
 group=com.rakuten.tech.mobile.miniapp
 name=miniappplatform
```

Comment:
I don't think you need to define `name`. If you leave it out then `name` will just get set to the module name (`miniapp`).

Reply:
The publishing part is to follow in the next PR. I didn't touch much there yet.
By the way, I was thinking to define a `rootProject.name` in settings.gradle instead.

Reply:
That's probably okay. I don't think `rootProject.name` is used for anything in the publishing config.